readme.md

```diff
@@ -397,20 +399,43 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Electric Guitar Specifications](https://github.com/gitfrage/guitarspecs) - Checklist for building your own electric guitar.
 
 
+## Business
+
+- [Open Companies](https://github.com/opencompany/awesome-open-company)
+- [Places to Post Your Startup](https://github.com/mmccaff/PlacesToPostYourStartup)
+- [OKR Methodology](https://github.com/domenicosolazzo/awesome-okr) - Goal setting & communication best practices.
+
+
+## Work
+
+- [Slack](https://github.com/matiassingers/awesome-slack) - Team collaboration.
+	- [Communities](https://github.com/filipelinhares/awesome-slack)
+- [Remote Jobs](https://github.com/lukasz-madon/awesome-remote-job)
+- [Productivity](https://github.com/jyguyomarch/awesome-productivity)
+- [Niche Job Boards](https://github.com/wfhio/awesome-job-boards)
+- [Programming Interviews](https://github.com/MaximAbramchuck/awesome-interview-questions)
+- [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows) - Productivity app for macOS.
```

Comment:
I would place this in "Development Environment" instead. It's not only for work.